src/simulator/base.py

```diff
@@ -0,0 +1,55 @@
+import object
+import position
+import shape
+import object_registry
+import numpy as np
+
+
+class Base(object.Object):
+    """
+    A physical station, primarily in charge of communicating with robots
+    and facilitating their navigation by transmitting their (estimated) distances with the base.
+    """
+
+    def __init__(self, oid: object.ObjectId, name: str, shape: shape.Shape, position: position.Position,
+                 owner_object: object.Object, registry: object_registry.ObjectRegistry,
+                 standard_deviation: float, time_using: float):
+        super().__init__(oid, name, shape, position, owner_object, registry)
+        self.standard_deviation = standard_deviation
+        self.__timer = 0
+        self.__time_using = time_using
+
+    def get_distance(self, position: position.Position) -> float:
+        """
+        It calculates and returns the distance between itself and the robot with some error.
```

Comment:
add some text saying that "In the current way, true distance is the distance between anchor points of the objects base and robot, i.e. probably their center points. This means that the true distance doesn't consider the dimensions of the objects."